docs/architecture/devops-for-aspnet-developers/actions-build.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Deploy an app to App Service - DevOps with .NET and GitHub Actions
+description: Deploy a .NET app to Azure App Service, the first step for DevOps with .NET and GitHub Actions.
+author: colindembovsky
+ms.date: 03/04/2021
+---
+# Deploy an app to App Service - DevOps with .NET and GitHub Actions
+
+[GitHub Actions](https://github.com/features/actions) allow you to automate workflows in response to events that are triggered in GitHub. Common workflows include Continuous Integration (CI) workflows, but Actions can be used to automate other processes such as sending out welcome emails when people join a repo.
+
+To explore how you can move code to cloud, you are going to build a GitHub Actions workflow file for the Simple Feed Reader application you have already deployed to Azure App Services.
+
+In this article you will:
+> [!div class="checklist"]
+
+> * learn the basic structure of a GitHub Action workflow YAML file
+> * Use a template to create a basic build workflow that build the .NET application and executes unit tests
+> * Publish the compiled application so that it is ready for deployment
+
+## Workflow Structure
+
+Workflows are captured in YML files and all have:
+
+- a `name`
+- a trigger, defined by an `on` section
+- one or more `job` sections, composed of one or more `steps`
+- optional attributes such as `environment` variables
+
+Jobs are run on _runners_. You can use _hosted runners_ which are spun up by GitHub for the duration of the workflow and then thrown away. Hosted runners are great because you don't have to maintain your own build infrastructure. For workflows that require specific build environment, or for running workflows on a private network, you can also use _private_ runners. To create a private runner, you can install the runner on any machine that supports .NET.
```

Comment:
```suggestion
Jobs are run on _runners_. You can use _hosted runners_, which are spun up by GitHub during the workflow and then thrown away. Hosted runners are great because you don't have to maintain your own build infrastructure. For workflows that require specific build environment, or for running workflows on a private network, you can also use _private_ runners. To create a private runner, you can install the runner on any machine that supports .NET.
```